ipatests/test_integration/test_installation.py

```diff
@@ -1304,6 +1304,22 @@ def test_backup_of_cs_cfg_is_created(self, server_cleanup):
         if original_cfg_content is not None:
             assert original_cfg_content == bcp_cfg_content
 
+    def test_pac_configuration_enabled(self):
```

Comment:
Move the test before `test_hostname_parameter` (otherwise it is called after a test that performs server_cleanup which uninstalls the master).